docs/installation.rst

```diff
@@ -25,6 +25,8 @@ We have checked the correct operation of OpenVSLAM and SocketViewer on Ubuntu 16
 
 :ref:`Docker <chapter-docker>` systems can be used instead of preparing the dependencies manually.
 
+If you're using Ubuntu, there are easy setup scripts in scripts/ubuntu. see scripts/ubuntu/Readme.md  
```

Comment:
Add it to docker.rst, not installation.rst. installation.rst is assumed to be referenced when docker is not used.

Reply:
Got it. I moved this comment in docker.rst.